src/main/java/de/uhd/ifi/se/decision/management/jira/filtering/impl/FilterExtractorImpl.java

```diff
@@ -222,10 +225,10 @@ private boolean checkIfTypeMatches(DecisionKnowledgeElement element) {
 					|| filterSettings.getDocumentationLocations().size() == 1 && filterSettings
 							.getDocumentationLocations().get(0).equals(DocumentationLocation.UNKNOWN)) {
 				// Check if the Status is filtered
-				if (filterSettings.getSelectedIssueStatus()
+				if (filterSettings.getSelectedStatus()
 						.contains(StatusPersistenceManager.getStatusForElement(element))) {
 					// Check if the Type of the Element is correct
-					if (checkIfTypeMatches(element) && checkIfElementMatchesTimeFilter(element)) {
+					if (checkIfKnowledgeTypeMatches(element) && checkIfElementMatchesTimeFilter(element)) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4631510)